examples/applications/plot_cyclical_feature_engineering.py

```diff
@@ -786,16 +788,17 @@ def periodic_spline_transformer(period, n_splines=None, degree=3):
     "Splines + polynomial kernel",
     "Gradient Boosted Trees",
 ]
-for ax, pred, label in zip(axes, predictions, labels):
-    ax.scatter(y.iloc[test_0].values, pred, alpha=0.3, label=label)
-    ax.plot([0, 1], [0, 1], "--", label="Perfect model")
-    ax.set(
-        xlim=(0, 1),
-        ylim=(0, 1),
-        xlabel="True demand",
-        ylabel="Predicted demand",
-    )
-    ax.legend()
+kind = ["actual_vs_predicted", "residual_vs_predicted"]
```

Comment:
The xticks of the actual vs predicted plot have too many digits.
I am wondering if we could instead share the x-axis as well and provide ticks from 0 to 1 with a step of 0.2.

Reply:
I think sharex has a conflict with the `PredictionErrorDisplay` API, namely the
```python
ax.set_aspect("equal", adjustable="datalim")
```
produces a
```python trace-back
RuntimeError: set_aspect(..., adjustable='datalim') or axis('equal') are not allowed when both axes are shared.
```

Reply:
I will set `adjustable="box"` but then the aspect is bit less pretty :(